### PR TITLE
[WIP] Add Drone CI package(s)

### PR DESCRIPTION
**Pull request by dellintosh**
This PR includes both the `drone-server` and `drone-agent` packages (since they work closely together).

Still need 
* [screenshots/images](https://github.com/mesosphere/universe#resourcejson) from the Drone maintainers.

### CODE REVIEW

File: repo/packages/D/drone-agent/0/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "packagingVersion": "3.0",
+    "description": "Drone is a Continuous Delivery platform built on Docker, written in Go.  This is the drone-agent; you MUST install the drone-server first.",
+    "framework": false,
+    "licenses": [
+        {
+            "name": "Apache license",
+            "url": "https://raw.githubusercontent.com/drone/drone/master/LICENSE"
+        }
+    ],
+    "maintainer": "brad@drone.io",
```
Comment (gtaylor) [inline repo/packages/D/drone-agent/0/package.json]: Might be better to find someone else who is actually willing and able to maintain this package. Brad is spread incredibly thin.

**Comment by ryadav88** [comment]
@dellintosh thanks. Can I also ask you to write short getting started doc for these packages so users don't have any issues installing and understanding how they work? 

You can create a PR against https://github.com/dcos/examples and refer other examples listed there. These examples will finally go here: https://dcos.io/docs/1.8/usage/tutorials/. Please follow the template of other examples here.

This is also now part of the package submission guidelines: https://github.com/mesosphere/universe#submit-your-package

File: repo/packages/D/drone-agent/0/package.json (hunk 1 above)
Comment (dellintosh) [inline repo/packages/D/drone-agent/0/package.json]: I've updated this to use my personal email address. ;)

**Comment by ryadav88** [comment]
@dellintosh is this ready for review?

**Comment by tobilg** [comment]
I just saw this, great work! Will love to try it soon! One thing I noticed was that the agents expose the docker.socket. As far as I know this will eventually interfere with the cgroup limits imposed by Mesos. I think Jenkins and the upcoming GitLab Runner packages are both using Docker-in-Docker solutions to work around this. 

cc @ryadav88

**Comment by tuxity** [comment]
When it will be merge ? :)

**Comment by ryadav88** [comment]
@dellintosh Ping. Is this ready for review?

**Comment by tobilg** [comment]
What about DinD @ryadav88?

File: repo/packages/D/drone-server/0/marathon.json.mustache [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+          "protocol": "tcp",
+          "servicePort": 0
+        }
+      ],
+      "privileged": false
+    },
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/var/run/docker.sock",
```
Comment (tuxity) [inline repo/packages/D/drone-server/0/marathon.json.mustache]: according to [this](http://readme.drone.io/admin/installation-guide/) the `drone-server` doesn't need the docker socket

**Comment by tuxity** [comment]
I've done a setup of it and it's working. 
But if the server go down, you lose your setup and you have to configure your repos again. I tried to use a volume mounted on `/var/lib/drone/` but it's not saved idk why

**Comment by ryadav88** [comment]
@dellintosh Any update here? If you'd like to come back to this at a later point, we should close it for now and reopen later

**Comment by ryadav88** [comment]
@dellintosh  Closing the PR due of inactivity. We can reopen it later if needed.

**Pull request closed without merging**